targets/targets.json

```diff
@@ -3366,7 +3366,7 @@
     "SAML21J18A": {
         "inherits": ["Target"],
         "core": "Cortex-M0+",
-        "macros": ["__SAML21J18A__", "I2C_MASTER_CALLBACK_MODE=true", "EXTINT_CALLBACK_MODE=true", "USART_CALLBACK_MODE=true", "TC_ASYNC=true"],
+        "macros": ["__SAML21J18A__", "I2C_MASTER_CALLBACK_MODE=true", "EXTINT_CALLBACK_MODE=true", "USART_CALLBACK_MODE=true", "TC_ASYNC=true", "CMSIS_VECTAB_VIRTUAL", "CMSIS_VECTAB_VIRTUAL_HEADER_FILE=\"cmsis_nvic.h\""],
         "extra_labels": ["Atmel", "SAM_CortexM0P", "SAML21"],
```

Comment:
release_versions is missing if you want to add support for OS 5. Also, why RTOS support is removed via .mbedignore?

Reply:
> why RTOS support is removed via .mbedignore?
as we don't use RTOS for our project and it was failing with: